### PR TITLE
Shows

### PR DESCRIPTION
**Pull request by itcomesinwaves**

### CODE REVIEW

File: client/src/routes/home/home.component.tsx [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
-    <h3>Top Haunts!</h3>
-    <CreepyCarousel />
-    <h3>Favorite horror books!</h3>
-    <CreepyCarousel />
-    <h3>chilling tales from around the web...</h3>
     <CreepyCarousel />
+    <br></br>
+    <h3>Top Haunts:</h3>
+    <Carousel2 /><br></br>
+    <h3>Favoirte tales of terror:</h3>
```
Comment (iamdestinos) [inline client/src/routes/home/home.component.tsx:40]: Typo

File: client/src/routes/movies-shows/movies.component.tsx [hunk 2]
```diff
@@ -14,19 +14,23 @@ const Movies = () => {
...
-  }; 
+  };  
 
   useEffect(() => {
     getMovies();
     setCurrentMoviesLoaded(true);
   }, []);
 
-console.log(currentMovies);
+console.log('in movies' ,currentMovies);
```
Comment (iamdestinos) [inline client/src/routes/movies-shows/movies.component.tsx:30]: Consider removing

File: client/src/routes/movies-shows/shows.component.tsx [hunk 3]
```diff
@@ -1,9 +1,40 @@
...
+          dispatch(setCurrentShows(data.slice(80, 150)));
+      })
+  };
+
+  useEffect(() => {
+    getShows();
+    setCurrentShowsLoaded(true);
+  }, []);
+
+  console.log('in showssss', currentShows);
```
Comment (iamdestinos) [inline client/src/routes/movies-shows/shows.component.tsx:26]: consider removing

**Review by iamdestinos — APPROVED** [review]
Lotta extraneous comments and consoles, but Approved

**Review by jimenezfede — APPROVED** [review]

**Pull request merged**